packages/frontend/src/components/molecules/Footer.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import Image from 'next/image';
+import Link from 'next/link';
+import LinkSection from '../atoms/LinkSection';
+import Subscribe from '../atoms/Subscribe';
+
+const links_sections = [
+  {
+    title: 'Product',
+    links: ['Employee database', 'Payroll', 'Absences', 'Time tracking'],
+  },
+  {
+    title: 'Information',
+    links: ['FAQ', 'Blog', 'Support'],
+  },
+  {
+    title: 'Company',
+    links: ['About us', 'Careers', 'Contact us', 'Lift Media'],
+  },
+];
+const social_media = [
+  {
+    element: 'in',
+  },
+  {
+    element: 'f',
+  },
+  {
+    element: <Image src="/icons/twitter-icon.png" width={18} height={18} alt="twt" />,
```

Comment:
u can't assume that an element could be a string then suddenly change it to an element, we have to stay consistent with the attribute types